Description: Minimise warnings in unix ijs code calling execvp
 execvp should really take const char *args,
 but linux is 'inconsistent' about this.
 We assume that the args are of this form in our code,
 so we'd rather have 1 warning on the call on such systems,
 rather than n about the setup on all systems.
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=7a7588c
Author: Robin Watts <robin.watts@artifex.com>
Forwarded: yes
Last-Update: 2019-08-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ijs_exec_unix.c
+++ b/ijs_exec_unix.c
@@ -57,7 +57,7 @@
   if (child_pid == 0)
     {
       int status;
-      char *argv[8];
+      const char *argv[8];
       int i = 0;
 
       close (fds_to[1]);
